package.json

```
@@ -2,7 +2,7 @@
   "name": "create-instantsearch-app",
   "version": "4.4.1",
   "license": "MIT",
-  "description": "⚡️ Build InstantSearch apps at the speed of thoughts",
+  "description": "⚡️ Build InstantSearch apps at speed of thoughts",
```

```suggestion
  "description": "⚡️ Build InstantSearch apps at the speed of thought",
```

This is how the quote usually is phrased in English